Packs/CommonScripts/Scripts/ExtractEmailFormatting/ExtractEmailFormatting.py

```diff
@@ -20,16 +20,21 @@ def extract_email(email_address: str) -> str:
     """
     email_address = email_address.lower()
 
+    if {"=", "?"}.issubset(set(email_address)):
+        # If we find these chars in a string it means the regex caught it as part of a url query and needs pruning.
+        email_address = extract_email_from_url_query(email_address)
+
     email_format = re.compile("[<(\[{\"\'.]*"
                               "(?:(?:\\\\|\^{3})u[a-f\d]{4})?"
                               "([\w.!#$%&'*+/=?^_`{|}~-]{1,64}"
                               "\[?@]?[\w.-]{1,255}(?:\[?\.]?"
                               "[A-Za-z]{2,}){1,2})", re.IGNORECASE)
 
-    try:
-        return re.findall(email_format, email_address)[0]
+    email_address = re.match(email_format, email_address)
 
-    except IndexError:
```

Comment:
re.match will always return, either a string or `None` so no need to catch `IndexError` it is done by the new `if` clause in line 35.